spec/linter-stylelint-spec.js

```diff
@@ -213,6 +217,34 @@ describe('The stylelint provider for Linter', () => {
     });
   });
 
+  describe('works with HTML files and', () => {
+    it('works with stylelint-config-standard', () => {
+      atom.config.set('linter-stylelint.enableHtmlLinting', true);
```

Comment:
You can probably just move this up into the `describe` (and get rid of the second one below).